models/schema.sql

```diff
@@ -13,10 +13,10 @@ DROP TABLE IF EXISTS Users;
 CREATE TABLE Users(
 	userID INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
     userLevelID INT NOT NULL,
-    name VARCHAR(50),
+    name VARCHAR(50) NOT NULL,
```

Comment:
Should any other attributes be NOT NULL? Or were you just working on Users?

Reply:
I think basically all of the attributes on all our tables should be `NOT NULL`, but let's discuss within the team

Reply:
I saw the comment in discord. I vote for making all NOT NULL.

Reply:
Yeah, I agree. I think all of those attributes are/should be necessary.